doc/backend/norms-everkuendung-api-v1.yaml

```diff
@@ -0,0 +1,153 @@
+        - Das Verfahren zum manuellen Übertragen der Daten, wenn die Verkündung wiederholt nicht angenommen wurde, muss noch definiert werden.
+        - Sollte die Antwort auf eine erfolgreiche Anfrage weitere Informationen enthalten oder reicht der Statuscode?
+        - Das Vorgehen und ggf. die Schnittstelle für die Übergangsphase, d.h. bevor die E-Gesetzgebung Daten aus dem LEA-Editor liefert, muss noch definiert werden.
+        - Eventuell wird es in Zukunft notwendig sein, statt einer synchronen Anfrage eine asynchrone Anfrage zu verwenden, damit das Backend größere Datenmengen besser verarbeiten kann. Es ist zu klären, ob wir die Schnittstelle direkt so gestalten wollen, oder ob das eine künftige Ausbaustufe sein wird.
+      security:
+        - ClientCredentials: [upload_verkuendung]
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                file:
+                  type: string
+                  format: binary
+                  description: ZIP-Archiv mit den Dokumenten
+                signature:
+                  type: string
+                  format: binary
+                  description: Digitale Signatur für das Archiv im PKCS#7-Format (.p7s)
+      responses:
+        "200":
+          description: Die Verkündung wurde erfolgreich an NeuRIS übermittelt.
+        "400":
+          description: |
+            Die Eingangsdaten sind fehlerhaft. Es kann sich um verschiedene Fehler handeln:
+
+            - Das ZIP-Archiv ist ungültig
+            - Die Signatur des Archivs fehlt oder ist ungültig
+            - Die Signatur der Dateien im Archiv fehlt oder ist ungültig
+        "401":
+          description: Fehlende Berechtigung, z.B. kann die Authentifizierung der Anfrage fehlen oder ungültig sein.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "409":
+          description: Duplikat erkannt. Eine Verkündung mit dieser ELI wurde bereits übermittelt.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "413":
+          description: Die Eingangsdaten überschreiten die maximale Dateigröße von 200mb.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "422":
+          description: |
+            Ungültige Eingangsdaten:
+
+            - Die XML-Dokumente weisen entweder XSD- oder Schematron-Fehler auf. Schematron-Warnungen führen nicht zu einem Fehler.
+            - Die XML-Dokumente liegen in einer nicht unterstützten Version von LegalDocML.de vor.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "500":
```

Comment:
Should we also add the `Retry-After` header for the response of the 500 error? Including info written in prosa before?